main.go

```diff
@@ -182,6 +184,9 @@ func (s *validTailnetSrv) run(ctx context.Context) error {
 		Ephemeral:  s.Ephemeral,
 		ControlURL: os.Getenv("TS_URL"),
 	}
+	if s.TsnetVerbose {
+		srv.Logf = log.Printf
```

Comment:
Just realized - we're not using [`slog.SetDefault`](https://pkg.go.dev/log/slog#SetDefault), but that would be really useful in order to get `log.Printf`-emitted logs to be the same format as the rest of the program's... How about this:

```suggestion
		slog.SetDefault(slog.Default())
		srv.Logf = log.Printf
```